api/server/call_logs.go

```diff
@@ -32,15 +32,11 @@ func writeJSON(c *gin.Context, callID, appID string, logReader io.Reader) {
 func (s *Server) handleCallLogGet(c *gin.Context) {
 	ctx := c.Request.Context()
 
-	app, err := s.datastore.GetApp(ctx, &models.App{Name: c.MustGet(api.App).(string)})
-	if err != nil {
-		handleErrorResponse(c, err)
-		return
-	}
-
+	appID := c.MustGet(api.AppID).(string)
+	appName := c.MustGet(api.App).(string)
```

Comment:
this is unfortunate really. before we lock API in maybe we get rid of the json field for app name in the log api? the app name is in the url to make this request now anyway. (this is a very minor breakage, I don't expect it hurts anyone) -- we're a bit over extended on this patch anyway, a TODO suffices if you agree

Reply:
agreed, will adjust this one